doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml

```diff
@@ -509,9 +509,30 @@ and faster to use a Transact-SQL `INSERT … SELECT` statement to copy the data.
 This value is incremented during the <xref:Microsoft.Data.SqlClient.SqlBulkCopy.SqlRowsCopied> event and does not imply that this number of rows has been sent to the server or committed.
 
 This value can be accessed during or after the execution of a bulk copy operation.
+
+This value can become negative if the number of rows exceelds int.MaxValue. If this may happen use the RowsCopied64 property.
```

Comment:
* `exceelds` -> `exceeds` (typo)
* `RowsCopied64` should have some kind of ref tag?
* I feel like this language should be stronger: "This value will wrap around and become negative if... Applications should use RowsCopied64 instead".

Reply:
Changed to:
`This value will wrap around and become negative if the number of rows exceeds int.MaxValue. Considering using the <xref:Microsoft.Data.SqlClient.SqlBulkCopy.RowsCopied64> property.`

If the tone needs to be stronger than that I'll leave the exact wording to the MS team.

Reply:
I like your rewording. As an app developer I would just always use RowsCopied64 now that it exists since there's no cost and real upside.